libraries/AP_BattMonitor/AP_BattMonitor_SMBus.h

```diff
@@ -4,6 +4,11 @@
 #include <AP_Param/AP_Param.h>
 #include <AP_Math/AP_Math.h>
 #include "AP_BattMonitor_Backend.h"
+#include <utility>
+
+extern const AP_HAL::HAL& hal;
```

Comment:
Hmmm I copied this in as the maxell/solo both used it (And I needed AP_HAL.h and figured there was no other reason for the weird declaration), but we appear to compile just fine with out it.... In fact it compiles if I remove it from the solo and maxell cpp files as well.... Removed.